src/components/hero/campaignHeroSubText.tsx

```diff
@@ -0,0 +1,30 @@
+
+import {
+	Link as DSLink,
+  HorizontalRule,
+	Text,
+  Spacer,
+} from "@nypl/design-system-react-components";
+
+const CampaignHeroSubText = () => {
+   return (  
+     <>
+     {/* To Do: make the link color blue: */}
+        <Text color="ui.typography.body">
+          This site is a living database with new materials added every day,
+          featuring prints, photographs, maps, manuscripts, streaming video, and
+          more.
+        </Text>
+        <Text color="ui.typography.body">
+        Our collections include some content that may be harmful or difficult to view. <DSLink href="https://digitalcollections.nypl.org/about#nypl_harmful_content_statement">Learn more</DSLink>
+        </Text>
+        <HorizontalRule/>
+        <Text color="ui.typography.body">
+        Featured Image:
```

Comment:
Btw, I'm not sure if you got a chance to add the `{" "}` code here for the extra space. The comment thread is no longer appearing, so just reiterating in case that got lost.

Reply:
oops, I think I resolved it by accident. thanks for the additional set of eyes.